src/scalarstats.jl

```diff
@@ -340,7 +340,7 @@ zscore!{T<:AbstractFloat,U<:Real,S<:Real}(X::AbstractArray{T}, μ::AbstractArray
 
 
 """
-    zscore(X[, μ, σ])
+    zscore(X, [μ, σ])
```

Comment:
I did it this way to signify that if you give one you have to give both. I saw this syntax looking through some of the docstrings for base functions. But if it's nonstandard I can go with your suggestion.